test/e2e/commands.js

```diff
@@ -0,0 +1,46 @@
+const path = require('path');
+
+/*
+function flatten (arr) {
+  try {
+    const allData = arr.reduce((result, fileData) => {
+      if (Array.isArray(fileData)) {
+        result.push(...fileData);
+      } else if (fileData instanceof Object) {
+        result.push(fileData);
+      }
+      return result;
+    }, []);
+    cy.wrap(allData).as('allData');
+  } catch {
+    cy.task('log', 'Issue with flatten');
+  }
+}
+*/
+
+const loadManifestFiles = () => {
+  try {
+    const manifest = 'language_data_manifest.js';
+    const allFileData = [];
+    cy.fixture(manifest).as('manifest_data').then((data) => {
+      data.forEach((item) => {
+        const name = path.parse(item).name;
+        cy.fixture(item).as(`${name}Data`).then((data) => {
+          allFileData.push(data);
+        });
+      });
+    }).then(() => {
+      // Potentially flatten data as with FW data.js
```

Comment:
Flattening nested arrays is much easier now.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/flat

```js
[[1,2,3,4],5,6,[[7,8,9],10]].flat(Infinity)
```
It is very much easier to sort and filter the data if it's flattened.
In adapt we add the file path to __path__ and the index for each path at __index__:

https://github.com/adaptlearning/adapt-contrib-core/blob/master/js/data.js#L114C9-L114C30

https://github.com/adaptlearning/adapt-contrib-core/blob/852ef1269dcdfc220361d500c473c9042ffa65f4/js/data.js#L147-L148

Reply:
@oliverfoster best practice according to Cypress is to have custom commands in a commands file, I think the config file is loaded too early to be able to set aliases, but I will double check that.